mtags/src/main/scala/scala/meta/internal/metals/Docstrings.scala

```diff
@@ -39,37 +39,26 @@ class Docstrings(index: GlobalSymbolIndex) {
     cache.get(symbol) match {
       case Some(value) =>
         if (value == EmptySymbolDocumentation) Optional.empty()
-        else Optional.of(value)
+        else if (value.docstring.nonEmpty) Optional.of(value)
```

Comment:
`Optional.of` suggests `value` can be a `null` but `value.docstring.nonEmpty` would fail in such case...